app/src/main/java/rick_and_morty/di/RickAndMortyApiRemoteDataSource.kt

```diff
@@ -1,12 +1,17 @@
 package rick_and_morty.di
 
 import retrofit2.http.GET
+import retrofit2.http.Path
 import retrofit2.http.Query
 import rick_and_morty.data.model.RickAndMortyResponseDto
 import rick_and_morty.data.api.ApiConstants
+import rick_and_morty.data.model.CharacterResultsDto
```

Comment:
why is this class in the DI package?

Reply:
It is same as RickAndMortyResponseDto, and needed to get return Type